README.md

```diff
@@ -1,37 +1,18 @@
-<h1 align="center">
-  <a href="">
-    <img src="/src/assets/movies.svg" alt="Project Banner Image">
-  </a>
-</h1>
-
 # Movie Site Project
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
-## Getting Started with the Project
-
-### Dependency Installation & Startup Development Server
+This is a project for creating a movie site that displays popular movies and their details. 
 
-Once cloned, navigate to the project's root directory and this project uses npm (Node Package Manager) to manage its dependencies.
-
-The command below is a combination of installing dependencies, opening up the project on VS Code and it will run a development server on your terminal.
-
-```bash
-npm i && code . && npm run dev
-```
+The project uses the The Movie Database (TMDb) API to fetch movie data.
 
 ### The Problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+The challenge for this project was to implement React Router to create routes for each movie's details page. We used React.js to create the site and fetch data from the TMDb API. We also used CSS to style the site and make it responsive.
+
+If we had more time, we could add more features to the site, such as ability to filter movies by genre.
 
 ### View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+Netlify link: 
 
-## Instructions
+https://mm-movie-browser.netlify.app/movie/575264
```

Comment:
This link isn't connected to the actual page..( /movie/... is direct to page not found)